Samples.Swift/Samples.Swift/ViewControllers/CheckoutViewController/Views/TotalCell.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TotalCell.swift
+//  paypalcheckout-iOS-swift-sample
+//
+//  Created by Jax DesMarais-Leder on 3/10/21.
+//
+
+import UIKit
+
+class TotalCell: UITableViewCell {
+  let totalLabel = UILabel()
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
```

Comment:
right now if we tap on this cell, the cell is highlighted, but there's no action on tap. Maybe we can add `selectionStyle = .none` for this cell I think